platform/feature-api/api/pod-api/src/default.ts

```diff
@@ -166,13 +166,22 @@ export class DefaultPod implements Pod {
             },
         };
     }
-    /**
-     * The [[Network]] interface. See [[Network]] for the description.
-     */
 
-    get network(): Network {
+    get endpoint(): Endpoint {
         return {
-            httpPost(url: string, body: string, contentType?: string, authorization?: string) {
+            send(
+                endpointId: string,
+                payload: string,
+                contentType?: string,
+                authorization?: string
+            ): Promise<void> {
+                throw new Error("Not implemented");
```

Comment:
Probably left for later, but a real implementation would allow testing of these things...